docs/EDID to Receiver Caps Mapping.md

```diff
@@ -202,16 +202,20 @@ This value MUST be transformed into `urn:x-nmos:cap:format:color_sampling` with
 
 The supported color subsampling formats in the CTA Extension Header ([CTA-861][CTA-861] section 7.5) indicate `YCbCr-4:2:2` and `YCbCr-4:4:4` support in addition to `RGB`.
 
-YCbCr 4:2:0 Capability Map Data Block ([CTA-861][CTA-861] section 7.5.11) shows which timings support `YCbCr-4:2:0` in addition to subsampling formats listed in the CTA Extension Header. These timings MUST contain `YCbCr-4:2:0` within the possible values for `urn:x-nmos:cap:format:color_sampling` with associated Constraint Set.
+YCbCr 4:2:0 Capability Map Data Block ([CTA-861][CTA-861] section 7.5.11) shows which video modes support `YCbCr-4:2:0` in addition to subsampling formats listed in the CTA Extension Header. Constraint Sets associated with these video modes MUST contain `YCbCr-4:2:0` within the possible values for `urn:x-nmos:cap:format:color_sampling`.
 
-YCbCr 4:2:0 Video Data Block ([CTA-861][CTA-861] section 7.5.10) marks timings as supporting only `YCbCr-4:2:0`. Constraint Set associated with these timings MUST have `urn:x-nmos:cap:format:color_sampling` limited to `YCbCr-4:2:0`.
+YCbCr 4:2:0 Video Data Block ([CTA-861][CTA-861] section 7.5.10) marks video modes as supporting only `YCbCr-4:2:0`. Constraint Sets associated with these video modes MUST have `urn:x-nmos:cap:format:color_sampling` limited to `YCbCr-4:2:0`.
 
 ### Color component depth
 
 _Color Bit Depth_ of _Video Input Definition_ described in [E-EDID A2][E-EDID] section 3.6.1 MUST be transformed into `urn:x-nmos:cap:format:component_depth` and MUST be added to each Constraint Set.
 
 Vendor-Specific Data Block ([CTA-861][CTA-861] section 7.5.4) SHOULD be transformed into `urn:x-nmos:cap:format:component_depth` if contains related information.
 
+### Colorspace
+
+Colorimetry Data Block ([CTA-861][CTA-861] section 7.5.5) SHOULD be transformed into `urn:x-nmos:cap:format:colorspace` if contains related information.
```

Comment:
"if contains related information" ... do you mean "with related information" ?

Reply:
Originally, I meant "if the Data Block is present and contains related information". Maybe the following will express both intentions.
```suggestion
Colorimetry Data Block ([CTA-861][CTA-861] section 7.5.5) SHOULD be transformed into `urn:x-nmos:cap:format:colorspace` with related information if present.
```